ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/pderesources.properties

```
@@ -1301,6 +1301,7 @@ UpdateClasspathResolution_label=Update the classpath and compliance settings
 UpdateExecutionEnvironment_label=Update the execution environment based on JRE on the classpath
 UpdateClasspathJob_task = Update classpaths...
 UpdateClasspathJob_title = Updating Plug-in Classpaths
+AddMatchingVersion_RequireBundle = Set latest available matching version
```

What do you think about the following?
```suggestion
AddMatchingVersion_RequireBundle = Require latest available version
```